### ISSUE TITLE
Collision does not instantly register

### ISSUE THREAD

**Issue by mikekoscinski** [mikekoscinski/cycling-game#18]
![image](https://user-images.githubusercontent.com/59475481/91341412-7b281300-e7a7-11ea-80a0-77ead72ed2df.png)

Collision logic appears flawed. Does not register collision until oncoming object has reached the opposite side of the biker.

**Comment by mikekoscinski** [comment]
This error only persists on the hosted (via firebase) version of the game. Can not replicate when running the code on my local machine.

**Comment by mikekoscinski** [comment]
Error is not present on mobile (iOS).

Have only been able to replicate error on browser (running Brave via macOS). Again, does not persist when I run my code locally - only when I use the actual hosted version (via Firebase) at pokecycle.com.

However, this issue does not occur every time I run a game session via pokecycle.com. It occurs sporadically. The type of the oncoming object (50/50 chance Object 0 is kabuto/omanyte) does not appear to correlate with the error's presence.

I reviewed performance recordings of the live site (pokecycle.com) in the DevTools console. Script execution does spike precisely when the initial collision occurs, as I would expect. Further, when the error does occur, the browser tab's favicon page refresh indicator begins to fire as soon as an initial collision occurs.

I think this is a firebase latency issue. The site is hosted with firebase's free tier; latency is to be expected.

Will continue to watch this but I think it will occasionally persist so long as I am using a free tier of firebase. Closing issue.
